hack/build.sh

```diff
@@ -9,6 +9,16 @@ declare -a VERSIONS=(2.4)
 OS=$1
 VERSION=$2
 
+function squash { 
+  # install the docker layer squashing tool
+  easy_install --user pip
```

Comment:
In fact you don't even need to install `pip`, just `easy_install --user docker-scripts==0.3.2` directly. Will save another line + time.

I guess it's time to go to bed now :)